sale_operating_unit_sequence/models/operating_unit.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class OperatingUnit(models.Model):
+    _inherit = "operating.unit"
+
+    so_sequence_id = fields.Many2one(
```

Comment:
I would prefer `sale_sequence_id`

Reply:
I changed `so_sequence_id` to `sale_sequence_id`.